perception/multi_object_tracker/src/data_association/successive_shortest_path/successive_shortest_path.cpp

```diff
@@ -47,10 +47,6 @@ void SSP::maximizeLinearAssignment(
   const std::vector<std::vector<double>> & cost, std::unordered_map<int, int> * direct_assignment,
   std::unordered_map<int, int> * reverse_assignment)
```

Comment:
```suggestion
  std::unordered_map<int, int> * reverse_assignment, const bool sparse_cost = true)
```

With the Notation in the original code, how about this? ( Need .hpp changes )